poetry/utils/exporter.py

```diff
@@ -94,7 +94,7 @@ def _export_requirements_txt(
                 dependency.marker = package.marker
 
                 line = "{}".format(package.source_url)
-                if package.develop:
+                if package.develop and package.source_type in ["directory", "url"]:
```

Comment:
We only want directory packages here, I think. As far as I know url packages cannot be editable.

Reply:
Good catch :+1: Fixed it.